frontend/src/pages/Account.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import '../styles/App.scss';
+import { updateMe } from '../actions/users';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchMe } from '../actions/users';
+
+//Account page for update user's info
+class AccountPage extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.onChangeEmail = this.onChangeEmail.bind(this);
+    this.onChangeNickName = this.onChangeNickName.bind(this);
+    this.state = { showresult: false };
+  }
+
+  componentDidMount() {
+    this.props.fetchMe();
+  }
+
+  onChangeEmail(e) {
+    const state = this.props.user;
+    state[e.target.name] = e.target.value;
+    this.setState(state);
+  }
+
+  onChangeNickName(e) {
+    const state = this.props.user;
+    state[e.target.name] = e.target.value;
+    this.setState(state);
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const data = this.state;
+    this.setState({ showresult: true });
```

Comment:
there's no guarantee that the request was successful here, better place to trigger a status message would be in the 'action', perhaps via sending a toast upon success. I'll show you an example of that later